ractor.rb

```diff
@@ -1,95 +1,101 @@
 # Ractor is an Actor-model abstraction for Ruby that provides thread-safe parallel execution.
 #
-# Ractor.new can make a new Ractor, and it will run in parallel.
+# Ractor.new makes a new ractor, which can run in parallel.
```

Comment:
```suggestion
# Ractor.new makes a new \Ractor, which can run in parallel.
```

This will allow us to be specific about the class, but not create a link in RDoc which is unnecessary here.